stable/ambassador/templates/deployment.yaml

```diff
@@ -120,7 +120,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.hostIP
-            {{- if and .Values.pro.enabled (gt .Values.pro.image.tag "0.6.0") }}
+            {{- if and .Values.pro.enabled (semverCompare .Values.pro.image.tag "0.6.0") }}
```

Comment:
Shouldn't this be `semverCompare ">0.6.0" .Values.pro.image.tag`, same as above?

Reply:
🤦‍♂